Students/30421/Papp - Szentannai G. Gergo/Assignment 1/1.1/src/main/Main.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * 
+ */
+package main;
+import java.util.Scanner;
+/**
+ * @author gergo_000
+ *
+ */
+public class Main {
+
+	/**
+	 * @param args
+	 */
+	public static void main(String[] args) {
+		
+		System.out.println("Enter max value:");
+		Scanner in = new Scanner(System.in);
+		int max = in.nextInt()+1; // remove +1 if max should not be included
+		
+		int sum=0;
+
+		for(int i = 1; i<max;i++){
+			if ((i%3==0) || (i%5==0)){
```

Comment:
this can be done a lot more elegantly. Check [this post](http://math.stackexchange.com/questions/9259/find-the-sum-of-all-the-multiples-of-3-or-5-below-1000)